### PR TITLE
Additional changes to the config

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

[description here]

## Checklist

Not all may apply:

- [ ] EPvalidator.xml has been updated
- [ ] Tests (and test files) have been updated
- [ ] Documentation has been updated
- [ ] Changelog has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI (checked comparison artifacts)

### CODE REVIEW

**Comment by shorowit** [comment]
Next time we merge something to master that affects results, we can follow up with a commit here to make sure it works correctly.

File: workflow/tests/compare.rb [hunk 1]
```diff
@@ -26,10 +26,13 @@
       results[key]['rows'] = CSV.read(filepath)
     else
       puts "Could not find #{filepath}."
-      next
     end
   end
 
+  if (not results[base].keys.include?('rows')) || (not results[feature].keys.include?('rows'))
+    next
+  end
```
Comment (shorowit) [inline workflow/tests/compare.rb:34]: @joseph-robertson What is this change for?

Comment (joseph-robertson) [reply workflow/tests/compare.rb:34]: @shorowit Basically, it allows handling the case where a feature branch adds or removes a results csv file.

File: Changelog.md [hunk 2]
```diff
@@ -243,4 +243,4 @@ __Bugfixes__
 
 ## OpenStudio-HPXML v0.7.0 Beta
 
-- Initial beta release
\ No newline at end of file
+- Initial beta release
```
Comment (joseph-robertson) [inline Changelog.md:246]: @shorowit Does it need to be a commit that changes results?

Comment (shorowit) [reply Changelog.md:246]: I'm testing that it doesn't reflect the changes that were just made to the master branch.

**Pull request merged**